### DISCUSSION TITLE
Kickoff 2022 (3h)

### DISCUSSION THREAD

**Discussion by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#2] [Meetings]
3h pub @g105b @richardbirkin

Kick off meeting to plan how to break this project down to bring real value as a side project.

### Introduction

Greg and I met with big bits of paper and some wine. 

We stopped Trackshift development in Autumn 2020, right as we were onboarding alpha users. There was too much going on in my life and the world and I wasn't making good decisions or doing good work. 

Since then, there has not been a day where I haven't thought about Trackshift. Greg and I have been waiting for our worlds, calendars and brains to align again. That time seems to be now!

We started with Domain Driven Design questions:

### What is the value?

Answer 1: Simple, easy to use + understand, aggregation of music royalties data for **Artists**. 

This extends to **Record Labels** (a user who will have data for more than one **Artist**) - making the distribution of royalties to **Artists** easier and more transparent.

![WhatsApp Image 2022-10-06 at 9 41 12 AM](https://user-images.githubusercontent.com/115098450/194266679-f1360617-9678-4a56-9834-94282e1dc4a6.jpeg)

From this we talked about:

- The value of _seeing_ before _registering_
- Gathering edge cases (where PRS data or copyright registrations were wrong, negatively affecting one artist or more) by survey
- Survival Budget HUD to help artists with financial planning and/or money-based applications

As well as getting a bit ahead of ourselves with (quickly terminated) talk of:

- Auth
- Currency Conversion

### What action are we taking?

- Fully utilise Github
- Develop completely in the open
- Avoid technical debt (no committing to any UI framework like we used to)
- Prioritise and commit to ideas (Design Sprint Philosophy)

![WhatsApp Image 2022-10-06 at 9 41 12 AM (1)](https://user-images.githubusercontent.com/115098450/194269887-b199d223-1202-40b8-be30-cb467e9d9b86.jpeg)

### What is next?

- Set up clean project locally
- Start talking about features on Github
 
### Our sequence for talking about new features:

1. What value are we trying to add?
2. What does that look like? 
- 2a: write automated tests to prove the value is being added.
- 2b: sketches/screenshots
3. Develop it

**Comment by g105b** [comment]
Step 2a: write automated tests to prove the value is being added.

**Reply by richardbirkin** [reply]
added
